includes/content-distribution/class-outgoing-author.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Newspack Network author distribution.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack_Network\Content_Distribution;
+
+use Newspack_Network\User_Update_Watcher;
+use WP_Error;
+use WP_Post;
+
+/**
+ * Class to handle author distribution.
+ *
+ * Every time a post is distributed, we also send all the information about the author (or authors if CAP is enabled)
+ * On the target site, the plugin will create the authors if they don't exist, and override the byline
+ */
+class Outgoing_Author {
+
+	/**
+	 * Gets the user data of a WP user to be distributed along with the post.
+	 *
+	 * @param int|WP_Post $user The user ID or object.
+	 *
+	 * @return WP_Error|array
+	 */
+	public static function get_wp_user_for_distribution( $user ) {
```

Comment:
I don't think we need a whole class for this. WDYT of moving this to the `Outgoing_Post` class?

It's similar to other methods we have there that are meant to build the payload, like `get_processed_post_content()`, `get_post_taxonomy_terms()`, and `get_the_post_thumbnail_url()`.